### PR TITLE
initial version of tutorial on typescript

### PR DESCRIPTION
**Pull request by abhutta0**
Sp23 version

### CODE REVIEW

File: tutorials/week1-typescript.md [hunk 1]
```diff
@@ -0,0 +1,1058 @@
...
+```
+
+*   _Usage_: 
+It lets you work with large integers without running into rounding errors. While the number type can only represent whole numbers it can hold numbers larger than 2<sup>53</sup> – 1. 
+
+### String
+
+String is the set of all strings and the things you can do with them like concatenate (+), slice (.slice),etc. 
+
+Example:
```
Comment (vivekananda-reddy) [inline tutorials/week1-typescript.md:117]: Examples for "any" were provided here instead of a string

Comment (shashwatrathod) [reply tutorials/week1-typescript.md:117]: Do you want to add a commit to this branch correcting this mistake? Let me know, else I can take it up

**Review by vivekananda-reddy — COMMENTED** [review]
Finished review and updated my changes

**Comment by abhutta0** [comment]
When others have reviewed, you can confirm. I will then merge to main

**Pull request merged**